manuscripts2/latex_template/activity/github_issues.tex

```diff
@@ -0,0 +1,14 @@
+Following chart shows activity in GitHub Issues data source. It is based on comparing the number of opened and closed tickets grouped by the time interval. A ticket is equivalent to an Issue in GitHub terminology.
```

Comment:
Maybe it should be "the following": https://www.google.es/amp/s/dictionary.cambridge.org/amp/british-grammar/following-or-the-following

Reply:
@valeriocos I think it is `opened` instead of `open`. That table shows the number of issues that were submitted in a time period and the number of issues that the maintainers were able to close for the same time period.